maint_tools/vendor_lazy_loader.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
maybe add a README to `externals/_lazy_loader` to explain the file shouldn't be modified and can be updated using this script?